src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1398,8 +1409,7 @@ impl Cage {
                     0
                 }
                 (F_DUPFD, arg) if arg >= 0 => {
```

Comment:
we probably need to add the check to see if fd and arg are equal to each other here like as in dup/dup2